### PR TITLE
Helpful hours screen

### PR DESCRIPTION
**Pull request by ttran0012**
I changed the module in order to make the item appears correctly. If the item doesn't appear correctly I did create a fork. 
npx pactch-package react-native-paper

if you can figure out another solution to fix this. That would be great.

### CODE REVIEW

File: components/HelpfulHoursTab/AcademicOffice.jsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+                console.log(err);
+            })
+    }, []);
+
+    return (
+        <View style = {styles.contentContainer}>
+        <List.Accordion 
+            children = {{children: 0}}
+            style = {styles.accordion}
+            title = "Academic Office"
```
Comment (Falzaro) [inline components/HelpfulHoursTab/AcademicOffice.jsx:37]: It looks like the files under components/HelpfulHoursTab all look more or less the same. Let's make the code more reusable.

File: components/HelpfulHoursTab/AcademicOffice.jsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+const AcademicOffice = () => {
+    const [academicOffice, setAcademicOffice] = useState([]);
+    // useState for List Accordions
+    const [expanded, setExpanded] = useState(false); // set this to false, don't want to open automatically
+    const handlePress = () => setExpanded(!expanded);
+
+    useEffect(() => {
+        // get AcademicOffice data from firestore
+        const docRef = doc(db, "helpful hours" , "academic office");
```
Comment (Falzaro) [inline components/HelpfulHoursTab/AcademicOffice.jsx:22]: You don't have to get a single document at a time from the collection "helpful hours". Firebase also has a function called "getDocs" to get all documents. You can map out the array of docs to generate a list of accordions.

**Comment by Falzaro** [comment]
image

I don't know if you are seeing this too, but there's still that left white space issue when the drop downs are expanded.

**Comment by ttran0012** [comment]
Case close. Re-updated code

**Pull request closed without merging**